### PR TITLE
Delete unpopular casks.

### PR DESCRIPTION
**Pull request by reitermarkus**
Delete all casks with less than 100 installs.

### CODE REVIEW

**Comment by reitermarkus** [comment]
For the remaining ones > 100, I'd say we check whether or not they have been updated in the last few months.

File: Casks/opal.rb
Comment (razvanazamfirei) [inline Casks/opal.rb:1]: This was added recently.

File: Casks/logitune.rb
Comment (razvanazamfirei) [inline Casks/logitune.rb:1]: Also added relatively recently (Feb 2023)

File: Casks/obsbot-webcam.rb
Comment (razvanazamfirei) [inline Casks/obsbot-webcam.rb:1]: Just realized I did not migrate this. Would be in favor of keeping it. https://github.com/Homebrew/homebrew-cask-drivers/issues/3414#issuecomment-1507345586

**Comment by reitermarkus** [comment]
@razvanazamfirei, please migrate the ones you mentioned (and possibly others you encounter that were recently added). Will rebase this once that's done.

**Comment by razvanazamfirei** [comment]
@reitermarkus, should be done once #3448 is merged.

**Review by p-linnane — APPROVED** [review]

**Review by bevanjkay — APPROVED** [review]
Merging this should reveal any casks that are worth moving in to `homebrew-cask` 👍

**Comment by toanju** [comment]
Any suggestion how I shall treat an unpopular one that I am using?

**Comment by reitermarkus** [comment]
@toanju, refer to https://github.com/Homebrew/homebrew-cask-drivers/issues/3414#issuecomment-1537881618.

**Comment by tomtastic** [comment]
```
Error: No formulae or casks found for "pallotron-yubiswitch".
```
Hmm, thats annoying, was it really so unpopular it got removed entirely?

**Comment by p-linnane** [comment]
> ```
> Error: No formulae or casks found for "pallotron-yubiswitch".
> ```
> 
> 
>     
>   
> 
> Hmm, thats annoying, was it really so unpopular it got removed entirely?

We are deprecating this repository. If there is something that was removed that you use, please refer to https://github.com/Homebrew/homebrew-cask-drivers/issues/3414#issuecomment-1537881618.

**Comment by tomtastic** [comment]
So folk now have to create up to 150 new PRs to migrate them individually, rather than you migrating them in one go ?
That feels like it sucks.

**Comment by p-linnane** [comment]
No. The odds are very high that most of these will not have replacement PR's opened because they are not used.

**Pull request merged**